components/ActionsDropdown/ActionsDropdown.tsx

```diff
@@ -120,3 +169,17 @@ ActionsDropdownArrow.defaultProps = {
 };
 
 ActionsDropdownArrow.displayName = 'ActionsDropdownArrow';
+
+/**
+ * ActionsDropdownButtonClose
+ */
+ export const ActionsDropdownButtonClose: React.FC<IBoxComponentProps> = withMooskinContext((props) => {
```

Comment:
no need to create this kind of component if we're not exposing it on a library level. use StyledActionsDropdownButtonClose directly and apply the necessary props as here.